person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    true if is_of_age? || @parent_permission
+  end
```

Comment:
- [ ] [optional] Nice application, it would be better if you even shorten it:

`hint`
`def can_use_services?`
    `is_of_age?() || @parent_permission`
 `end`

Reply:
Thanks so much for the clue